src/customerCalls/components/CallFormWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const CallFormWrapper = styled.div`
+    font-family: ${props => props.theme.fonts.primary};
+    box-sizing: border-box;
+    max-height: 80px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    @media only screen and (max-width: 415px) {
```

Comment:
as in mutiple examples above, use variable for max-width

Reply:
done